tests/kafkatest/__init__.py

```diff
@@ -23,4 +23,4 @@
 # Instead, in trunk, the version should have a suffix of the form ".devN"
 #
 # For example, when Kafka is at version 0.9.0.0-SNAPSHOT, this should be something like "0.9.0.0.dev0"
-__version__ = '0.10.0.0.dev0'
+__version__ = '0.10.0.1.dev0'
```

Comment:
@gwenshap @ijuma 
Yes the leading part of the version should match version in `gradle.properties`, so it looks like we still want `0.10.0.0`